packages/ip2locationio/ip2locationio.1.0.0/opam

```diff
@@ -0,0 +1,45 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "IP2Location.io OCaml module to get geolocation and WHOIS data"
+description:
+  "IP2Location.io OCaml module allows user to query for an enriched data set based on IP address and provides WHOIS lookup api that helps users to obtain domain information. "
+maintainer: ["IP2Location.com"]
+authors: ["IP2Location.com"]
+license: "MIT"
+tags: ["ip2locationio" "geolocation"]
+homepage: "https://github.com/ip2location/ip2location-io-ocaml"
+doc: "https://github.com/ip2location/ip2location-io-ocaml"
+bug-reports: "https://github.com/ip2location/ip2location-io-ocaml/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "3.4"}
+  "lwt"
+  "cohttp"
+  "cohttp-lwt-unix"
```

Comment:
You probably need the following to run the tests on the CI

```suggestion
  "cohttp-lwt-unix"
  "tls-lwt" {with-test}
```

(there is a choice between `tls-lwt` and `lwt_ssl` and I'm suggesting `tls-lwt` here for no particular reason)